tests/test_PsfexPsf.py

```diff
@@ -284,6 +286,19 @@ def testPsfexDeterminerTooFewStars(self):
         with self.assertRaisesRegex(RuntimeError, "Failed to determine"):
             psf, cellSet = self.psfDeterminer.determinePsf(self.exposure, psfCandidateListShort, metadata)
 
+    def testPsfDeterminerChangeNormField(self):
+        """Test the psfDeterminer with a different normalization field."""
+        self.setupDeterminer(self.exposure, normField="base_CircularApertureFlux_12_0_instFlux")
```

Comment:
Should you set this to some other value entirely, so that once we switch the default to 12 we don't have to remember to change it here?